# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/other/Platform.cpp
# Commit: 2f75e063b5f9
# Full Hash: 2f75e063b5f903dec563f53045a46e2ba56bef2c
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 1: Provide an argument to PlatformCaretMoveEvent specifying whether the event was caused by user input. r=eeejay
#   
#   This is needed in order to support find in page on Android using remote caret events instead of virtual cursor change events.
#   
#   Depends on D192641
# ==============================================================================

diff -r 70924ad2b7e3 -r 2f75e063b5f9 accessible/other/Platform.cpp
--- a/accessible/other/Platform.cpp	Wed Nov 29 05:51:34 2023 +0200
+++ b/accessible/other/Platform.cpp	Wed Nov 29 03:53:19 2023 +0000
@@ -27,7 +27,8 @@
 void a11y::PlatformCaretMoveEvent(Accessible* aTarget, int32_t aOffset,
                                   bool aIsSelectionCollapsed,
                                   int32_t aGranularity,
-                                  const LayoutDeviceIntRect& aCaretRect) {}
+                                  const LayoutDeviceIntRect& aCaretRect,
+                                  bool aFromUser) {}
 
 void a11y::PlatformTextChangeEvent(Accessible*, const nsAString&, int32_t,
                                    uint32_t, bool, bool) {}